fish_prompt.fish

```diff
@@ -213,6 +213,10 @@ function prompt_status -d "the symbols for a non zero exit status, root and back
     end
 end
 
+function available -a name -d "Check if a function or program is available."
```

Comment:
Oh, yeah sorry. This was because I actually don't use OMF myself and thus couldn't use this theme without manually copying the `available` function. I forgot to remove it before sending the pull request. I shall remove it in a new one.